notebooks/en/quantize_stable_diffusion_with_quanto.ipynb

```diff
@@ -0,0 +1,777 @@
+{
```

Comment:
<p>this can be a simple markdown text and not a header (applies for many steps below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>